Kanban/Repositories/StoryRepository.cs

```diff
@@ -24,6 +24,8 @@ public async Task<List<Story>> GetAllStories()
 
         public async Task<bool> CreateStory(Story story)
         {
+            story.Id = default;
```

Comment:
What is default?

Again, most likely the issue is that you use the story as a view model. But obviously if you do not want to have a CreateStoryRequest... :)